.github/workflows/ci-format.yml

```diff
@@ -36,4 +36,35 @@ jobs:
             -s n \
             --ignore marl_benchmark,examples,scenarios,docs,manager_pb2_grpc.py,worker_pb2_grpc.py \
             --msg-template='{path}: line {line}: {msg_id}: {msg}' \
-            ./smarts ./envision
\ No newline at end of file
+            ./smarts ./envision
+
+  test-types:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    container: huaweinoah/smarts:v0.4.18-minimal
```

Comment:
can we just use "latest" for the version here?

Reply:
The container `huaweinoah/smarts:v0.4.18-minimal` only has the necessary apt packages and sumo installed. All other smarts python files are installed dynamically during CI. No change appears to be needed here for now.

Whereas the container `huaweinoah/smarts:latest` contains apt packages, sumo, and pre-installed smarts files. This is not suited for our CI jobs.

See the difference between the two docker files here: https://github.com/huawei-noah/SMARTS/tree/standard-obs/utils/docker .

Reply:
ok